test/integration/targets/virt_net/aliases

```diff
@@ -0,0 +1,5 @@
+destructive
+shippable/posix/group1
+skip/freebsd
+skip/osx
+needs/privileged
```

Comment:
Does this test even work under docker? If not, remove this.

Reply:
It does.